toonz/sources/toonz/filebrowsermodel.cpp

```diff
@@ -1087,7 +1087,10 @@ void DvDirModelRootNode::refreshChildren() {
       std::wstring roothDir = projectRoot.getWideString();
       DvDirModelSpecialFileFolderNode *projectRootNode =
           new DvDirModelSpecialFileFolderNode(
-              this, L"Project root (" + roothDir + L")", projectRoot);
+              this,
+              L"*" + projectRoot.withoutParentDir().getWideString()
+              + L" (" +roothDir + L")",
```

Comment:
Is 'roothDir' intentional or should this (for clarity) be 'rootDir'?
Should work either way just curious if the extra letter h has a specific purpose.